src/plugins/intel_cpu/src/nodes/kernels/registers_pool.hpp

```diff
@@ -9,11 +9,12 @@
 #include "ie_common.h"
 #include "utils/cpu_utils.hpp"
 #include <utility>
+#include "stack_allocator.hpp"
 
 namespace ov {
 namespace intel_cpu {
 
-using namespace dnnl::impl::cpu;
+namespace x64 = dnnl::impl::cpu::x64;
```

Comment:
As far as I can see only `x64::cpu_isa_t` is used. M.b. it does make sense to use an alias for this type on the `RegistersPool` level:
```cpp
class RegistersPool {
public:
    using cpu_isa_t = dnnl::impl::cpu::x64::cpu_isa_t;
```
So, within this header it can be accessed just as follows `RegistersPool::cpu_isa_t`
What do you think?

Reply:
We also have other references to x64 namespace in this header like x64::cpu_isa_traits and cpu_isa_t enum members.

Reply:
Did not see an elegant solution here, since the whole class is intended to be used with dnnl::impl::cpu::x64::cpu_isa_t members, and we do not have a possibility to import enumeration to the class (it has been introduced in c++20). I guess we should think about wrapping some dnnl entities on jit_base level.